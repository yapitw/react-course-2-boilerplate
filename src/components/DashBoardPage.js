import React from 'react';

const DashboardPage = () => (
  <div className="content-container">
    Dashboard page content
  </div>
);

export default DashboardPage;