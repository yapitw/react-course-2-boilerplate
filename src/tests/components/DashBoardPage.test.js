import React from 'react';
import { shallow } from 'enzyme';
import DashBoardPage from '../../components/DashBoardPage';

test('should render DashBoardPage correctly', () => {
  const wrapper = shallow(<DashBoardPage />);
  expect(wrapper).toMatchSnapshot();
});
