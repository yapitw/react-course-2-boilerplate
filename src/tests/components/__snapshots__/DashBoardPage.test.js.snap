// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashBoardPage correctly 1`] = `
<div
  className="content-container"
>
  Dashboard page content
</div>
`;
